### ISSUE TITLE
Crash after a few minutes

### ISSUE THREAD

**Issue by brendanlong** [jangernert/FeedReader#889]
From @narga on #476

I've tried the newest version and found the Unicode characters are display correctly but, the app always crash after several minutes. I ran it from the commandline then has:
`(feedreader:32102): GLib-DEBUG: 15:11:45.284: posix_spawn avoided (fd close requested)
An error occured (SIGSEGV)
The reason is likely a null reference being used.
Note: no file path and line numbers can be retrieved. Are you sure you added -g -X -rdynamic to valac command line?

```
#1  <unknown>   in 'g_thread_join'
    at /usr/lib/libglib-2.0.so.0
#2  <unknown>   in 'g_closure_invoke'
    at /usr/lib/libgobject-2.0.so.0
#3  <unknown>   in 'g_signal_emit_valist'
    at /usr/lib/libgobject-2.0.so.0
#4  <unknown>   in 'g_signal_emit'
    at /usr/lib/libgobject-2.0.so.0
#5  <unknown>   in 'g_main_context_dispatch'
    at /usr/lib/libglib-2.0.so.0
#6  <unknown>   in 'g_main_context_iteration'
    at /usr/lib/libglib-2.0.so.0
#7  <unknown>   in 'g_application_run'
    at /usr/lib/libgio-2.0.so.0
#8  <unknown>   in 'feed_reader_main_main'
    at feedreader
#9  <unknown>   in '__libc_start_main'
    at 
#10  <unknown>   in '_start'
    at feedreader
```

**Comment by brendanlong** [comment]
@Narga Can you confirm that this is in the latest version (v2.8.2), and you're using the TT-RSS plugin?

**Comment by Narga** [comment]
Yes, I'm using the latest version of FeedReader and TT-RSS plugin activated on my hosted TT-RSS website. 
I switched to flatpak version, it's not crash but stuck at sync. FeedReader didn't sync for a long time, I tried to restart it sometimes but it's still not sync.

**Comment by bilelmoussaoui** [comment]
@Narga can you try to delete your database and start again? just to see what is making FeedReader stuck at synching

**Comment by Narga** [comment]
I followed your guide then it works. But after several days, it happened again. My TT-RSS has more than 200k items, is it the cause of this problem? The sync process take a lot of time.

**Comment by brendanlong** [comment]
Unfortunately the backtrace isn't very helpful and I can't reproduce this. I wouldn't be surprised if there was something wrong with our TT-RSS plugin. I added a bunch of TT-RSS fixes recently though (193ea1291c6135f5357e6a9a340f323b49defbad, 7fcb774f766d265d576724ed1f45947d87957c0c, a52474a7fea023abc6ff9b873db6a0067f091254), so it would help if you could try the latest version (v2.9.0).

I'm going to close this because I think the TT-RSS plugin should be working now, but please let me know if you still see this problem in v2.9.0.

**Comment by Narga** [comment]
:+1:
